server/config.js

```diff
@@ -147,4 +147,6 @@ module.exports = {
     autostart: get('SCHEDULED_JOBS_AUTOSTART', 'no') === 'yes',
     overlapTimeout: get('SCHEDULED_JOBS_OVERLAP', 5000),
   },
+
+  continueCaToRoFeatureFlag: get('CONTINUE_CA_TO_RO_FEATURE_FLAG', 'yes'),
```

Comment:
generally nicer to coerce this into a boolean, like above: 
`autostart: get('SCHEDULED_JOBS_AUTOSTART', 'no') === 'yes'`

then you'd be able to:
```
if (!continueCaToRoFeatureFlag) {
        return null
 }
```

I think the logic is the wrong way round in your check? if the flag is enabled then you want to use the new feature.
So in dev you want to use the real check - whereas in production you'd want to always return null?

Reply:
I have changed the code as per above but it doesn't seem logical to me that we should negate the continueCaToRoFeatureFlag in caService